src/main/resources/assets/thaumicboots/lang/en_US.lang

```diff
@@ -102,10 +102,16 @@ itemGroup.Thaumic Boots=Thaumic Boots
 keybinding.speedtoggle=Modulate Speed Boost
 keybinding.jumptoggle=Modulate Jump Boost
 keybinding.omnitoggle=Toggle Omnidirectional Movement
+keybinding.inertiacancellingtoggle=Toggle Cancelling Flight Inertia
 
 thaumicboots.jumpEffect=Jump Boost:
 thaumicboots.speedEffect=Speed Boost:
 thaumicboots.omniEffect=Omni Mode:
+thaumicboots.inertiaCancellingEffect=Inertia Cancelling Mode:
```

Comment:
That's a lot of text, I'd just change it to "Inertia State:"

Reply:
How about "Inertia Canceling:" , shortens it but still keeps it consistent with how it is in code.